groups/prometheus/locals.tf

```diff
@@ -0,0 +1,18 @@
+data "vault_generic_secret" "secrets" {
+  path = "team-${var.team}/${var.account_name}/${var.region}/${var.repository_name}"
+}
+
+locals {
+  discovery_availability_zones = join(",", list("${var.region}a", "${var.region}b", "${var.region}c"))
+  dns_zone_name = data.vault_generic_secret.secrets.data["dns_zone_name"]
+  internal_cidrs = values(data.terraform_remote_state.networking.outputs.internal_cidrs)
+  placement_subnet_availability_zones = [for subnet in values(data.aws_subnet.placement) : lookup(subnet, "availability_zone")]
+  placement_subnet_name_patterns = jsondecode(data.vault_generic_secret.secrets.data["placement_subnet_name_patterns"])
+  vpc_id = data.aws_vpc.vpc.id
+  vpc_name = data.vault_generic_secret.secrets.data["vpc_name"]
```

Comment:
I'd tweak the syntax here as per [this](https://github.com/companieshouse/concourse-terraform/pull/155/commits/7e89f1b6ef71a4caa5797cf871401244faa8908e)